prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/InstallCommand.java

```diff
@@ -285,4 +303,31 @@ public void verify() {
             }
         }
     }
+
+    private void printListOfProfiles() throws Exception {
+        Set<String> featurePackNames = KnownFeaturePacks.getNames();
+        for(String channelNames: featurePackNames){
+            KnownFeaturePack knownFeaturePack =  KnownFeaturePacks.getByName(channelNames);
+            final ChannelPrinter channelPrinter = new ChannelPrinter(console);
+            console.println("---channels----");
```

Comment:
Is it fine if I am printing channels as caption?

Reply:
Maybe use more descriptive headers. Something like `Included feature packs:`, `Subscribed channels:`